src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -211,6 +215,22 @@ public void register(boolean slimefun) {
 				this.permission = SlimefunPlugin.getItemCfg().getString(this.id + ".required-permission");
 				this.noPermissionTooltip = SlimefunPlugin.getItemCfg().getStringList(this.id + ".no-permission-tooltip");
 				
+				if (this.hint != null) {
+					SlimefunPlugin.getItemCfg().setDefaultValue(this.id + ".hint", this.hint);
+
+					ItemMeta im = this.getItem().getItemMeta();
+					List<String> lore = im.getLore();
+
+					lore = (lore == null) ? new ArrayList<>() : lore;
+
+					lore.add("");
+					lore.addAll(SlimefunPlugin.getItemCfg().getStringList(this.id + ".hint"));
+					lore.replaceAll(s -> ChatColor.translateAlternateColorCodes('&', s));
```

Comment:
Should use the utility methos in CS-CoreLib2. `replaceAll(ChatColors::color)`

Reply:
Thanks, wasn't aware of this method